apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/BBCodingSystems.java

```diff
@@ -54,6 +53,12 @@ public static class MCS {
 
     public static final String STATUS_DATE = MCS_BASE + "/status-date";
 
+    public static final String ICN = MCS_BASE + "/icn";
+
+    public static final String PROC_CODE = MCS_BASE + "/proc-code";
```

Comment:
@KyRobbins are PROC_CODE and MOD_PREFIX still in use?

Reply:
They do not seem to be, I'll remove those 👍